downstream/archive/archived-assemblies/dev-guide/assembly-migrate-ansible-versions.adoc

```diff
@@ -24,7 +24,7 @@ The Ansible _Porting Guide_ is a series of documents that provide information on
 [role="_additional-resources"]
 == Additional resources
 
-* link:https://docs.ansible.com/ansible/devel/porting_guides/porting_guide_2.9.html#ansible-2-9-porting-guide[Ansible 2.9]
+* link:https://docs.ansible.com/projects/ansible/devel/porting_guides/porting_guide_2.9.html#ansible-2-9-porting-guide[Ansible 2.9]
```

Comment:
Should this link to the top level porting guide page for Ansible Core?
https://docs.ansible.com/projects/ansible/latest/porting_guides/core_porting_guides.html